synapse/app/homeserver.py

```diff
@@ -656,11 +658,13 @@ def start_generate_monthly_active_users():
         )
 
     start_generate_monthly_active_users()
-    if hs.config.limit_usage_by_mau or hs.config.mau_stats_only:
+    if hs.config.run_background_tasks and (
+        hs.config.limit_usage_by_mau or hs.config.mau_stats_only
+    ):
         clock.looping_call(start_generate_monthly_active_users, 5 * 60 * 1000)
     # End of monthly active user settings
 
-    if hs.config.report_stats:
+    if hs.config.run_background_tasks and hs.config.report_stats:
```

Comment:
`start_phone_stats_home` and `performance_stats_init` might need to actually run for each process? So this might have been an incorrect change.

Reply:
I'm still not 100% sure about this.

Reply:
I think we should only do it on one process.